src/main/java/whisper/WhisperException.java

```diff
@@ -0,0 +1,46 @@
+package whisper;
+import java.util.*;
+
+public class WhisperException extends Exception {
+    public WhisperException(String err) {
+        super(err);
+    }
+
+    // Invalid command
+    public static WhisperException unknownCommand() {
+        return new WhisperException("\nInvalid Command, please try again.\nWe only support commands such as:" +
+                "\n- todo [description]\n- event [description] /from [start] /to [end]" +
+                "\n- deadline [description] /by[time]\n- mark [taskID]\n- unmark [taskID]\n- delete [taskID]\n");
+    }
+
+    // tasks has more than 100 task
```

Comment:
Your comments really help improve readability and understanding, good job!! 💯

Reply:
thanks!